src/test/java/wooteco/subway/acceptance/StationAcceptanceTest.java

```diff
@@ -35,26 +33,24 @@ void createStation() {
                 .post("/stations")
                 .then().log().all()
                 .extract();
-
         // then
         assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
         assertThat(response.header("Location")).isNotBlank();
     }
 
     @DisplayName("기존에 존재하는 지하철역 이름으로 지하철역을 생성한다.")
```

Comment:
DisplayName으로 메서드의 행위가 잘 표현되어 있네요. 브라운이 잘 표현해주셨네요(?)ㅋㅋㅋ

행위 이외에 해당 테스트를 통해 검증하려는 결과도 표현되면 좋겠어요. 

```suggestion
    @DisplayName("기존에 존재하는 지하철역 이름으로 지하철역을 생성시 400을 응답한다.")
```

이러면 기존의 방식과 어떤 차이가 있을까요?

Reply:
적용해보니 기존의 방식은 테스트하는 메서드의 행위만을 표현했다면 변경한 방식은 그 행위를 통해 받을 수 있는 결과까지 표현할 수 있는 것 같습니다. 이후에 작성할 때도 이처럼 `행위 + 결과`를 나타보면 좋을 것 같다는 생각이 드네요 ㅎㅎ

Reply:
수정을 잘 해주셨네요! 행위와 결과가 같이 표현되면 Displayname만 읽더라도 내부의 테스트에 어떤 Given, When, Then이 존재할지 머릿속에 그려지기도 해요. 그러면 코드를 파악하는 데에 걸리는 시간이 줄어들구요. 
이뿐 아니라 바니가 테스트를 작성하기 전에 해당 테스트에서 어떤 내용을 담아야 할지를 한글로 적어보기에 실수를 줄일 수 있어요 😊